shell/common/api/electron_api_v8_util.cc

```diff
@@ -103,6 +103,16 @@ void RequestGarbageCollectionForTesting(v8::Isolate* isolate) {
       v8::Isolate::GarbageCollectionType::kFullGarbageCollection);
 }
 
+// This causes a fatal error by creating a circular extension dependency.
```

Comment:
Can we wrap this in a `DCHECK_ENABLED` check or something so it is literally only exposed in a testing build?